README.md

```diff
@@ -26,7 +26,16 @@ The latest official **installers** and **portable versions** for macOS, Windows
 
 ### macOS
 
-It is recommended to install *Stretchly* with [Homebrew](https://brew.sh/) by running `brew update && brew install --cask stretchly`, see [Application Signing](#application-signing). When upgrading, run `brew update && brew upgrade --cask`. Don't forget to Quit Stretchly, first.
+It is recommended to install *Stretchly* with [Homebrew](https://brew.sh/) by running:
+```zsh
+brew update && brew install --cask stretchly
+```
+See [Application Signing](#application-signing). 
```

Comment:
can we move this up? 

It is recommended to install Stretchly with Homebrew  (see Application Signing) by running: